python/core/qgsconditionalstyle.sip

```diff
@@ -25,10 +25,11 @@ class QgsConditionalLayerStyles
   public:
     QgsConditionalLayerStyles();
 
-    /** Returns row styles.
-     * @return row styles
-     */
-    QList<QgsConditionalStyle> rowStyles();
+    QList<QgsConditionalStyle> rowStyles() const;
+%Docstring
+ Returns row styles.
+ @return row styles
```

Comment:
Yes I have some work to do on documentation. I have to add this note too `@note added in QGIS 3.0` to new functions.